framework/requirements.txt

```diff
@@ -95,6 +95,3 @@ Werkzeug==3.0.1
 xmltodict==0.12.0
 yarl==1.7.0
 zipp==3.3.2
-content_size_limit_asgi
-uvicorn==0.24.0.post1
-content_size_limit_asgi==0.1.5
```

Comment:
Why are these dependencies being removed?  If their removal is required, we should open a new issue to generate the packages again. Last update: https://github.com/wazuh/wazuh/issues/20795

Reply:
They are duplicated, I removed them during the development but forgot to rollback the changes and open an issue before opening the pull request.